bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/PowerControlPower.java

```diff
@@ -32,6 +33,10 @@
 
     @Override
     public void onReceiveCommand(String channelId, LGWebOSHandler handler, Command command) {
+        if (RefreshType.REFRESH == command) {
+            handler.postUpdate(channelId, handler.getSocket().isConnected() ? OnOffType.ON : OnOffType.OFF);
+            return;
```

Comment:
Maybe suppress this return and add else jus after like you do in the class MediaControlPlayer or VolumeControlMute ?

Reply:
what follows is not a complete if else cascade.  I find it more readable to jump out early